internal/aez/general.go

```diff
@@ -70,6 +70,29 @@ const (
 	PM RID = 30
 )
 
+func (rid RID) String() string {
```

Comment:
This file "general.go" was intended to be related to the "General status housekeeping structure" (STAT), RID is related to CCD data?

Reply:
I think we should move STAT to a `stat.go` or at least I find it confusing finding it.

Reply:
RID is the transparency data equivalent of SID and so yes it is related to CCD but not only.

Reply:
neither of them (RID/SID) belongs in this file.

Reply:
OK to rename the file to stat.go but then, the files containing htr,pwr etc. also should be renamed.

Reply:
will make issue for renaming instead since it may be awkward right now that we are actively coding in the aez package.